src/cli/task-docs.ts

```diff
@@ -1,4 +1,5 @@
-import { isOutputTargetDocs } from '../compiler/output-targets/output-utils';
```

Comment:
when I was fiddling with build stuff I noticed this relative import from `src/compiler/` which was causing code from within the compiler bundle be also pulled into the `cli` bundle. I considered a bit how to address it - we could export the output targets related utils (and constants) from the compiler bundle, making this into something like

```ts
import { isOutputTargetDocs } from '@stencil/core'
```

but I thought that wasn't too great for a few reasons, mainly that it changes the public API of `@stencil/core`. I also thought just for organization, if the output target related utils are used across the codebase they should really live in the `src/utils` directory and therefore be importable from the `@utils` namespace. So I thought we should look at moving the functions to `src/utils/output-target.ts` and moving the constants to `src/utils/constants.ts`